### PR TITLE
Default to reset_on_send=True for ChatInterface

### PR DESCRIPTION
**Pull request by ahuang11**
Closes https://github.com/holoviz/panel/issues/5694

### CODE REVIEW

File: panel/chat/interface.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ class ChatInterface(ChatFeed):
     )
 
     reset_on_send = param.Boolean(
-        default=False,
+        default=True,
```
Comment (MarcSkovMadsen) [inline panel/chat/interface.py:83]: I believe the reference guide should be updated too then.

Comment (ahuang11) [reply panel/chat/interface.py:83]: I think the reference guide doesn't mention the default.

File: panel/chat/interface.py [hunk 2]
```diff
@@ -319,6 +319,7 @@ def _click_send(self, _: param.parameterized.Event | None = None) -> None:
                     file_name=active_widget.filename,
                 )
             # don't use isinstance here; TextAreaInput subclasses TextInput
+            print(self.reset_on_send)
```
Comment (MarcSkovMadsen) [inline panel/chat/interface.py:322]: Should be removed.

Comment (philippjfr) [inline panel/chat/interface.py:322]:
```suggestion
```

**Review by philippjfr — APPROVED** [review]

**Pull request merged**